api/v01/entities/vrp_result.rb

```diff
@@ -83,7 +83,7 @@ class VrpResultSolutionRoute < Grape::Entity
       expose :end_time, documentation: { type: Integer, desc: 'Give the actual end time of the current route if provided by the solver' }
       expose :geometry, documentation: { type: String, desc: 'Contains the geometry of the route, if asked in first place' }
       expose :initial_loads, using: VrpResultDetailQuantities, documentation: { is_array: true, desc: 'Give the actual initial loads of the route' }
-      expose :costs, using: VRPResultDetailedCosts, documentation: { desc: 'The impact of the current route within the solution cost' }
+      expose :cost_details, using: VRPResultDetailedCosts, documentation: { desc: 'The impact of the current route within the solution cost' }
```

Comment:
It is not an input field, if we don't let this version extend over months it's ok

Reply:
It is not input but it is output. As long as it is the public API input and output are the same importance. We will merge this as soon as possible to dev but the problem is dev->master... if it takes as long as the last time (10+ months) then your client will be long using the `costs` field.

Reply:
It is output but still public API. this will be merged soon but dev->master took 10 months last time. So if it takes that long then your client will be long using the "costs"